src/components/enable-push-notifications-drawer/EnablePushNotificationsDrawer.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useCallback } from 'react'
+
+import { StyleSheet, View } from 'react-native'
+
+import IconNotification from '../../assets/images/iconGradientNotification.svg'
+import IconHeart from '../../assets/images/iconHeart.svg'
+import IconRepost from '../../assets/images/iconRepost.svg'
+import IconRemix from '../../assets/images/iconRemix.svg'
+import IconExploreNewReleases from '../../assets/images/iconExploreNewReleases.svg'
+import IconFollow from '../../assets/images/iconFollow.svg'
+import IconCoSign from '../../assets/images/iconCoSign.svg'
+
+import Button from '../../components/button'
+import Drawer from '../../components/drawer'
+import Text from '../../components/text'
+
+// Importing directly from audius-client temporarily until
+// settings page is migrated because we still need push notification logic to work
+// on settings page and it doesn't necessarily make sense in common
```

Comment:
makes sense. lets capture this kind of dependency on linear or something

Reply:
Ok, noted in the settings page ticket